### PR TITLE
exercice about grep

### PR DESCRIPTION
**Pull request by camillemarini**
exercice for instructor training

### CODE REVIEW

File: 06-find.md [hunk 1]
```diff
@@ -481,6 +481,43 @@ about them."
...
+> containing a list of dates and associated number of this species, such as 
+> for rabbits:
+> 
+> ~~~
+> 2013-11-05,22
+> 2013-11-06,19
+> 2013-11-07,18
+> ~~~
+>
+> Help her by putting in the right reorder the following commands and pipes :
```
Comment (ashwinsrnth) [inline 06-find.md]: Just a tiny bit of re-wording needed here: "right order" would sound better

Comment (camillemarini) [inline 06-find.md]: thanks. I think it is clearer indeed :-)

File: 06-find.md [hunk 2]
```diff
@@ -481,6 +481,43 @@ about them."
 >
 > 4. None of the above.
 
+> ## Track a species {.challenge}
```
Comment (gvwilson) [inline 06-find.md]: Species

File: 06-find.md [hunk 3]
```diff
@@ -481,6 +481,43 @@ about them."
...
+> 
+> Do you remember Leah from the previous lesson? She has several hundred
```
Comment (gvwilson) [inline 06-find.md]: "Leah has several hundred..."

File: 06-find.md [hunk 4]
```diff
@@ -481,6 +481,43 @@ about them."
...
+> containing a list of dates and associated number of this species, such as 
+> for rabbits:
+> 
+> ~~~
+> 2013-11-05,22
+> 2013-11-06,19
+> 2013-11-07,18
+> ~~~
+>
+> Help her by putting in the right order the following commands and pipes :
```
Comment (gvwilson) [inline 06-find.md]: Put these commands and pipes in the right order to achieve this:

**Pull request merged**